README.md

```diff
@@ -1356,7 +1356,7 @@ Other Style Guides
     });
     ```
 
-  - [18.3](#18.3) <a name='18.3'></a> Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). Place no space before the argument list in function calls and declarations.
+  - [18.3](#18.3) <a name='18.3'></a> Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). As well as a space before the argument list in function calls and declarations.
```

Comment:
Only real change from `function()` to `function ()`

I can go either way on this, but I know sublime doesnt highlight `function()` as an actual function. Or at least it didnt when I used it